backend/bom-bom-server/src/main/java/me/bombom/api/v1/article/controller/ArticleController.java

```diff
@@ -21,9 +22,9 @@ public class ArticleController {
 
     private final ArticleService articleService;
 
-    @GetMapping("/{memberId}")
+    @GetMapping
     public Page<ArticleResponse> getArticles(
-            @PathVariable Long memberId,
+            @RequestParam Long memberId,
             @RequestParam(required = false) LocalDate date,
             @RequestParam(required = false) String category,
             @RequestParam(required = false, name = "sorted", defaultValue = "DESC") SortOption sortOption,
```

Comment:
p3

추후에  더 요소가 늘어날 것 같은데 이렇게 따로 모아서 관리하는 건 어떤가요?😎

```suggestion
            @ModelAttribute ArticleCondition condition,
```

```java
public record ArticleCondition(
    @RequestParam("memberId") Long memberId,
    @RequestParam(value = "date", required = false) LocalDate date,
    @RequestParam(value = "category", required = false) String category,
    @RequestParam(value = "sorted", defaultValue = "DESC") SortOption sortOption
) {}
```